src/components/Tx/SendTx/CurrencySelect.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Identicon from '../../Identicon'
+
+export default class CurrencySelect extends Component {
+  static displayName = 'CurrencySelect'
+
+  static propTypes = {
+    etherWallet: PropTypes.object,
+    onClick: PropTypes.func,
+    tokens: PropTypes.array
```

Comment:
I would define this as PropTypes.arrayOf() and define what the array inside should look like